testing/renderer-with-debug-ui.tsx

```diff
@@ -72,13 +79,14 @@ export const RendererWithDebugUI = ({
                         <Renderer
                             // @ts-expect-error - TS2322 - Type 'MutableRefObject<Renderer | null | undefined>' is not assignable to type 'LegacyRef<Renderer> | undefined'.
                             ref={ref}
-                            content={question.content}
+                            content={renderedQuestion.content}
                             images={question.images}
                             widgets={question.widgets}
                             problemNum={0}
                             apiOptions={controlledAPIOptions}
                             reviewMode={reviewMode}
                             strings={strings}
+                            answerless={answerless}
```

Comment:
What does the `answerless` prop do here? I don't see it defined in the `Renderer` prop types.

Reply:
Removed!